### DISCUSSION TITLE
The vulnerability CVE-2022-2989 has been fixed, but no specific tag denotes the patched version.

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [containers/podman#19065] [General]
Hello, we are a team researching the dependency management mechanism of Golang. During our analysis, we came across your project and noticed that you have fixed a vulnerability ([snyk references](https://security.snyk.io/vuln/SNYK-GOLANG-GITHUBCOMCONTAINERSPODMANV4LIBPOD-3114918), CVE: CVE-2022-2989, CWE: CWE-842, fix commit id: dce3d6ee9d4e6ae49b6bd34697173dd3b784e475). However, we observed that you have not tagged the fixing commit or its subsequent commits. As a result, users are unable to obtain the patch version through Go tool ‘go list’.

We kindly request your assistance in addressing this issue. Tagging the fixing commit or its subsequent commits will greatly benefit users who rely on your project and are seeking the patched version to address the vulnerability.

We greatly appreciate your attention to this matter and collaboration in resolving it. Thank you for your time and for your valuable contributions to our research.

**Comment by Luap99** [comment]
The linked CVE and commit do not match?!

CVE-2022-2989 was fixed in d82a41687e614d9ac8b2d169dee47fe226835e4c and that is included in all tags after 4.3.

**Reply by Silence-worker-02** [reply]
Thank you for your assistance. I encountered an issue with the snyk references, as it provided me with an incorrect commit as the patch instead of the actual fixing commit.
